webpack/private/webpack_bundle.bzl

```diff
@@ -398,21 +432,26 @@ def webpack_bundle(
 
         **kwargs: Additional arguments
     """
-    if entry_point and entry_points:
-        fail("Cannot specify both entry_point and entry_points")
-    if not entry_point and not entry_points:
-        fail("One of entry_point or entry_points must be specified")
 
-    _webpack_bundle(
+    webpack_configs = create_configs(
```

Comment:
I think it would be nice if this was just actions within the rule like before this PR, but that means the devserver needs to be more then just a thin wrapper over `js_run_devserver`. WDYT?

Reply:
Good like this IMO